backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,103 @@
+create table project
+(
+    project_id  bigserial not null primary key,
+    name        text      not null unique,
+    description text      null
+);
+
+create table test_methodology
+(
+    test_methodology_id bigserial     not null primary key,
+    name                text          not null unique,
+    description         text          null,
+    confluence_link     varchar(1000) null
+);
+
+-- Corresponds to an EXCEL file
+create table experiment
+(
+    experiment_id  bigserial                not null primary key,
+    project_id     bigint                   not null,
+    methodology_id bigint                   not null,
+    -- This is input from the user
+    name           text                     null,
+    meta           text                     null,
+    -- This is fallback name to show if name is not specified
+    raw_file_name  text                     not null,
+    upload_date    timestamp with time zone not null,
+
+    constraint belongs_to_project
+        foreign key (project_id) references project (project_id)
+            on update cascade
```

Comment:
Don't we want `on update no action` like in `has_methodology` because `project_id` is immutable? Also why not `restrict`?

Reply:
no action and restrict are +- the same, the are some quirks regarding deferred constraints in `restrict`

Reply:
Ok, whatever then.